<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HelpSettings</class>
 <widget class="QDialog" name="HelpSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pomoc i ustawienia</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="HelpInfoP">
     <property name="title">
      <string>Klawisze sterujące</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="HelpInfo">
        <property name="text">
         <string>Esc - Klawiatura
F1 - Informacje i ustawienia
F2 - Wczytaj zawartość
F3 - Zapisz zawartość
F4 - Pełny ekran
F5 - Start
F6 - Stop
F7 - Reset
F8 - Zerowanie
F9 - Magnetofon i pamięć
F11 - Jeden krok</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="SettingsP">
     <property name="title">
      <string>Ustawienia</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="3" column="0">
       <widget class="QLabel" name="VolumeSoundL">
        <property name="text">
         <string>Głośność generatora:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="VolumeBeepL">
        <property name="text">
         <string>Głośność sygnalizatora:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSlider" name="VolumeBeepS">
        <property name="maximum">
         <number>250</number>
        </property>
        <property name="value">
         <number>125</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="AudioTimerT">
        <property name="text">
         <string>20</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="AudioTimerL">
        <property name="text">
         <string>Okres timera (ms):</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSlider" name="VolumeSoundS">
        <property name="maximum">
         <number>250</number>
        </property>
        <property name="value">
         <number>125</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="AudioBufferL">
        <property name="text">
         <string>Długość bufora (smp):</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="AudioElementL">
        <property name="text">
         <string>Długość elementu (smp):</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="AudioBufferT">
        <property name="text">
         <string>7000</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="AudioElementT">
        <property name="text">
         <string>1500</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="ScreenNegative">
        <property name="text">
         <string>Negatyw</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="ScreenColors">
        <property name="text">
         <string>Kolory</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="DebugP">
     <property name="title">
      <string>Debugowanie</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QCheckBox" name="DebugReg0">
        <property name="text">
         <string>Pamięć</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="DebugSaveFile">
        <property name="text">
         <string>Do pliku na bieżąco</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="DebugReg1">
        <property name="text">
         <string>Rejestry ABCDEFHL główne</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="DebugTrace">
        <property name="text">
         <string>Rejestrowanie rozkazów</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="DebugReg3">
        <property name="text">
         <string>Rejestry IX,IY,SP,I,R</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="DebugReg2">
        <property name="text">
         <string>Rejestry ABCDEFHL dodatkowe</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
